docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,192 @@
-belongs to a business organization), the list of “all workspaces” itself needs to be exposed as an API object to an end
-user inside a workspace.  That workspace is “virtual” - it adapts or transforms the underlying source of truth for the
-object and potentially the schema the user sees.
+- the user may have access to different workspaces: a workspace that belongs to the user personally, or one that
+  belongs to a business organization. The user might want to list and manage their API objects from all workspaces in a
+  single call (e.g. single `kubectl get` invocation),
+- a service provider might want to get a list of all instances (from all users) of the API resource they provide,
+  so that they can do reconciliation, have insights into how their API is used, and more.
 
-## Index (e.g. Workspace Index)
+Virtual Workspaces are limited in terms of what they can see, they can usually see only selected resources (e.g.
+resources owned by that use or resources provided by a given service provider). It's important to note that virtual
+workspaces are **not** read-only, they can also be used to modify resources (for more details see the document linked
+below).
 
-An index is the authoritative list of a particular API in their source of truth across the system.  For instance, in
-order for a user to see all the workspaces they have available, they must consult the workspace index to return a list
-of their workspaces.  It is expected that indices are suitable for consistent LIST/WATCHing (in the kubernetes sense) so
-that integrations can be built to view the list of those objects.
+kcp provides a set of default virtual workspaces, but you can also build your own virtual workspaces. Virtual
+Workspaces are implemented in the Go programming language using the appropriate kcp library
+([`pkg/virtual/framework`](https://github.com/kcp-dev/kcp/tree/main/pkg/virtual/framework)), and then you run them
+along with kcp.
 
-Index in the control plane sense should not be confused with secondary indices (in the database sense), which may be
-used to enable a particular index.
+More information, including concrete examples and a list of frequently asked questions, can be found in the
+[Virtual Workspaces document](../workspaces/virtual-workspaces/).
+
+## Exporting/Binding APIs
+
+One of the core values of the kcp project is to enable providing APIs that can be consumed inside multiple logical
+clusters. This is done via an exporting/binding mechanism, i.e. via API Exports and API Bindings.
+
+The general workflow is:
+
+- Define your API resources through `APIResourceSchemes` similar to how you would define CustomResourceDefinitions
+  (CRDs)
+- Create `APIExport` in the same workspace/logical cluster with a list of `APIResourceSchemes` that you want to export
+- User can create `APIBinding` in their workspaces referring to your `APIExport`. This will result in API resources that
```

Comment:
```suggestion
- Users can create `APIBinding` in their workspaces referring to your `APIExport` (if you grant the permission). This will result in API resources that
```